openedx/features/pakx/lms/pakx_admin_app/views.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.response import Response
+from .permissions import CanAccessPakXAdminPanel
+
+
+@api_view(('GET',))
+@permission_classes([CanAccessPakXAdminPanel])
+def temp(request):
```

Comment:
this should be part of this PR. PR should not contain any dev testing code

Reply:
Done!